lms/templates/instructor/instructor_dashboard_2/cohort_management.html

```diff
@@ -27,7 +28,7 @@
             % for content_group in content_groups:
             {
                 id: ${content_group.id},
-                name: "${content_group.name | h}",
+                name: "${content_group.name | n, js_escaped_string}",
```

Comment:
This improper escaping was the cause of my bok choy failure with the injected string (and automated xss testing disabled).